10-JavaScript/fun-with-functions.md

```diff
@@ -42,6 +42,19 @@ sayItTwice("JS is OK!"); // "JS is OK!"
                          // "JS is OK!"
 ```
 
+##### Default Function arguments
```

Comment:
I think it'd make this header one size larger since this way is basically just bolded.